### PR TITLE
feat: here they come to snuff the rooster, oh yeah

### PR DESCRIPTION
**Pull request by idoshamun**
### Describe what this PR does

Ain't found a way to kill me yet
Eyes burn with stinging sweat
Seems every path leads me to nowhere
Wife and kids, household pet
Army green was no safe bet
The bullets scream to me from somewhere
Here they come to snuff the rooster, oh yeah
Yeah here come the rooster, yeah
You know he ain't gonna die
No, no, no oh, you know he ain't gonna die
Here they come to snuff the rooster, oh yeah
Yeah here come the rooster, yeah
You know he ain't gonna die, oh
No, no, no oh, you know he ain't gonna die
Walkin' tall machine gun men
They spit on me in my home land
Gloria sent me pictures of my boy
Got my pills 'gainst mosquito death
My buddy's breathing his dyin' breath
Oh God please won't you help me make it through
Here they come to snuff the rooster, oh yeah
Yeah here come the rooster, yeah
You know he ain't gonna die
No, no, no oh, you know he ain't gonna die

## Events

Did you introduce any new tracking events?
Don't forget to update the [Analytics Taxonomy sheet](https://docs.google.com/spreadsheets/d/18Lv7zXges9QfVX5VYL1a-Hyl0e1sQ3sLr0OK8YZWKXI/edit#gid=0)

Log the new/changed events below:

| Type   | event_name  | value |
|--------|-------------|-------|
| Change/New | event name  | extra: { ... } |

### On those affected packages:
- [ ] Have you done sanity checks in the webapp?
- [ ] Have you done sanity checks in the extension?
- [ ] Does this not break anything in companion?

### Did you test the modified components media queries?
- [ ] MobileL (420px)
- [ ] Tablet (656px)
- [ ] Laptop (1020px)

#### Did you test on actual mobile devices?
- [ ] iOS (Chrome and Safari)
- [ ] Android

WT-{number} #done

### CODE REVIEW

**Review by omBratteng — APPROVED** [review]

**Review by omBratteng — APPROVED** [review]
lgtm

File: packages/webapp/pages/roast.tsx [hunk 1]
```diff
@@ -0,0 +1,265 @@
...
+    ? () => onGenerateImage({ theme: themes[themeIndex].value })
+    : () => showLogin({ trigger: AuthTriggers.Roast });
+
+  return (
+    <>
+      <LazyImage
+        imgSrc="https://daily-now-res.cloudinary.com/image/upload/s--bLTH_SfU--/f_auto/v1707242870/public/Roast-emoji.png.png"
+        imgAlt="Devil emoji"
+        ratio="80%"
+        className="-mb-12 w-[340px]"
```
Comment (rebelchris) [inline packages/webapp/pages/roast.tsx]: Can we change this to rem?

**Comment by omBratteng** [comment]
image

**Review by omBratteng — APPROVED** [review]
lgtm again

**Review by rebelchris — APPROVED** [review]

**Review by kopancek — COMMENTED** [review]
Looks good to me, would like to see some tests for the new functionality :)

**Comment by davidercruz** [comment]
Happy Christian Bale

**Pull request merged**